tsconfig.build.json

```diff
@@ -1,4 +1,15 @@
 {
-    "extends": "./tsconfig",
```

Comment:
I think this file is exactly the same as tsconfig.json and tsconfig.test.json, no? I think they should be a bit different:

The config files should be different from one another. e.g., the include/exclude should be different, and for the test we don't need the declaration fies

```
// tsconfig.json

{
    "compilerOptions": {
        "module": "esnext",
        "esModuleInterop": true,
        "sourceMap": true,
        "declaration": true,
        "target": "ES5",
        "declarationDir": "dist/definitions",
        "lib": ["es6", "dom"],
        "moduleResolution": "node",
        "outDir": "dist"
    }
}

// tsconfig.build.json
{
    "extends": "./tsconfig",
    "include": ["src/*"],
    "exclude": ["node_modules", "src/**/*.spec.ts"]
}

// tsconfig.test.json
{
    "extends": "./tsconfig",
    "compilerOptions": {
      "declaration": false,
    }
    "include": ["src/**/*.spec.ts"] <- I'm not sure about this one with how Jest handle stuff
}
```